sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -38,49 +38,58 @@ static char addr_str[NG_IPV6_ADDR_MAX_STR_LEN];
 static ng_ipv6_addr_t *_add_addr_to_entry(ng_ipv6_netif_t *entry, const ng_ipv6_addr_t *addr,
                                           uint8_t prefix_len, uint8_t flags)
 {
+    ng_ipv6_netif_addr_t *free_entry;
```

Comment:
That was pretty much my train of thought ^^ I opted for tmp_addr, but I can change it to free_addr too. Let me just fix that segfault first...